arroyo/processing/strategies/run_task_with_multiprocessing.py

```diff
@@ -413,13 +428,22 @@ def __init__(
         max_batch_time: float,
         input_block_size: int,
         output_block_size: int,
+        resize_input_blocks: bool = False,
+        resize_output_blocks: bool = False,
+        max_input_block_size: Optional[int] = None,
+        max_output_block_size: Optional[int] = None,
```

Comment:
With so many extra parameters, this seems like it would be more complicated rather than less to figure out what to pass here

Reply:
yeah I just want to make this opt-in for now. I think long-term we should give  `input_block_size` and default, and enable auto-resizing by default. Then we still have a lot of parameters, but people don't _need_ to set any of them.

Right now they _need_ to configure `input_block_size`, and we can't give them a default.

Reply:
If the user knows what the max is supposed to be, what's the rationale for them to start with a smaller number at all, rather than just passing the max into `input_block_size` and `output_block_size`